### PR TITLE
protocol out

### PR DESCRIPTION
**Pull request by boucadair**

### CODE REVIEW

**Comment by dhruvdhody** [comment]
We have a parallel update in which the charter says - 

````
The development of solutions and protocol extensions is not within the scope. Further, the SRv6OPS WG will not adopt any work until the respective WG has finished WGLC.
````

Is that okay?

**Comment by boucadair** [comment]
I'm not sure to parse the last part of the sentence: 

"Further, the SRv6OPS WG will not adopt any work until the respective WG has finished WGLC."

Do you mean:  work that depends on other spec developed in another WG won't be adopted till that spec is WGLCed in that WG?

**Comment by dhruvdhody** [comment]
Yes, do you have better way to put it across?

**Comment by boucadair** [comment]
I'm not sure it is a good idea to add that restriction as (ideally) ops considerations should be worked in // when the spec is under active development.

**Comment by dhruvdhody** [comment]
The aim is to have this discussion in the WG where it is being developed
and not to have the discussion in two places. The worry that i heard from
several folks is there being conflicting guidance between spring and
srv6ops for instance on things in development....

**Comment by dhruvdhody** [comment]
This does not mean that agenda time won't be given to those topics, we can still use SRv6ops as a way to discuss operational impact but then take it as an individual input to the respective WG and we will not formally adopt I-D with operational guidance on tech that is still in development.

**Comment by boucadair** [comment]
Here is an attempt: 

"The development of solutions and protocol extensions is not within the scope of the SRv6OPS WG. Furthermore, the SRv6OPS WG won't consider adopting documents that discuss operational considerations and guidance for SRv6-related technologies that are still under development in other WGs (SPRING, 6MAN, for example). However, the SRv6OPS WG will welcome presentations to increase awareness to potential operational issues and seek for feedback from the operators community."

File: SRv6OPS-Charter.md [hunk 1]
```diff
@@ -9,7 +9,8 @@ The SRv6OPS WG focuses on the operational aspects of deploying and managing Segm
 * Provide advice on standardizing operational tools and mechanisms for SRv6 networks.
 
 ## Scope:
-The SRv6OPS WG focuses on best practices, deployment guidance, operational issues, new requirements, and potential operational experiments. Solutions and protocol extensions are not within its scope. The scope of the SRv6OPS WG includes:
+The SRv6OPS WG focuses on best practices, deployment guidance, operational issues, new requirements, and potential operational experiments.
+Development of new protocols and protocol extensions are not within the scope of the WG. The scope of the SRv6OPS WG includes:
```
Comment (boucadair) [inline SRv6OPS-Charter.md]:
```suggestion
The development of solutions and protocol extensions is not within the scope of the SRv6OPS WG. Furthermore, the SRv6OPS WG won't consider adopting documents that discuss operational considerations and guidance for SRv6-related technologies that are still under development in other WGs (SPRING, 6MAN, for example). However, the SRv6OPS WG will welcome presentations to increase awareness to potential operational issues and seek for feedback from the operators community.

The scope of the SRv6OPS WG includes:
```

**Pull request merged**